README.md

```diff
@@ -322,6 +323,20 @@ If you use [n] as Node.js version manager, please, set `SPACESHIP_NODE_DEFAULT_V
 | `SPACESHIP_NODE_DEFAULT_VERSION` | `` | Node.js version to be treated as default (for [n] support) |
 | `SPACESHIP_NODE_COLOR` | `green` | Color of Node.js section |
 
+### npm package version (`npm`)
```

Comment:
function identifier of the prompt section is shown in the parenthesis. So here instead of `npm` use the relevant function suffix (`npm_package_version`)